### PR TITLE
Write an upgrade guide

### PR DESCRIPTION
**Pull request by jubianchi**

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# atoum upgrade guide
+
+## From 2.x to 3.x
+
+### Runtime
+
+atoum `3.x` requires **PHP `>= 5.6.0`.
```
Comment (agallou) [inline UPGRADE.md]: mettre `*PHP*` ?

File: UPGRADE.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    	$this->testedInstance->doSomethingAndThrow();
+    })
+;    
+```
+
+### Reports
+
+Some reports have been moved to a dedicated extension: [`atoum/reports-extension`](https://github.com/atoum/reports-extension).
+
+If you are using of of those reports, consider using the extension or simply remove them as they are not part of atoum anymore:
```
Comment (agallou) [inline UPGRADE.md]: "one of those reports" ?

File: UPGRADE.md (hunk 1 above)
Comment (Hywan) [inline UPGRADE.md]: Or nothing. Why PHP should receive a specific semantics here?

Comment (jubianchi) [inline UPGRADE.md]: Because I want to highlight the requirements.

**Pull request merged**